stories/Atom/Base-typography/Mark/mark.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../../assets/scss/mixins";
+@import "../../../assets/scss/breakpoints";
+@import "../../../assets/scss/variables";
+
+mark {
+    @extend %paragragh-font-20;
+    background: $color-light-yellow;
+    padding: 2px;
+    // font-size: 14px;
+    // line-height: 19px;
```

Comment:
@pallavikadam-srijan remove the commented code

Reply:
Done